nextflow.config

```
@@ -176,12 +184,13 @@ dag {
 
 manifest {
     name            = 'nf-core/phyloplace'
-    author          = 'Daniel Lundin'
+    author          = """Daniel Lundin"""
     homePage        = 'https://github.com/nf-core/phyloplace'
-    description     = 'Performs phylogenetic placement with EPA-NG'
+    description     = """Performs phylogenetic placement with EPA-NG"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=21.10.3'
+    nextflowVersion = '!>=22.10.1'
     version         = '1.0dev'
```

since this is a PR to master, dev shouldnt be here, because next step would be the release, right?
```suggestion
    version         = '1.0.0'
```
rather than accepting the suggestion better use nf-core bump-version, see first step of https://nf-co.re/developers/release_checklist#steps-to-release